frontend/src/core/plural/selectors.js

```diff
@@ -24,7 +24,7 @@ export function _getPluralForm(pluralForm: number, entity: ?Entity): number {
  */
 export const getPluralForm: Function = createSelector(
     pluralSelector,
-    entities.selectors.getSelectedEntity,
+    (state) => entities.selectors.getSelectedEntity(state),
```

Comment:
Why are you making this change?

Reply:
As I explained in the commit message, this change is necessary to avoid circular import errors.

In subsequent PRs this import would fail:
```
import { useOnDiscard } from 'core/utils';
```
with a very cryptic error message. Delaying the evaluation of `entities.selectors` solved the issue.

Reply:
Ah, sorry, I didn't look at the commit messages. Thanks! :)